about.html

```diff
@@ -4,25 +4,106 @@
+            <div class="social-media-icon">
+                <img src="./res/fb.svg" alt="fb">
+                <img src="./res/twitter.svg" alt="Twitter">
+            </div>
+            <div class="top-header-options">
+                <a href="#">English</a>
+                <a href="#">My Page</a>
+                <a href="#">Log Out</a>
+            </div>
+        </div>
+    </header>
     <nav id="navbar" class="nav-bar">
+
         <div id="hamMenu" class="ham-menu"><img src="./res/mobile_menu.svg" alt="Ham img"></div>
-        <div id="logo"><a href="./index.html"><img src="./res/logo.png" alt="Logo Image"></a></div>
+        <div id="logo"><a href="./index.html"><img src="./res/logo.svg" alt="Logo Image"></a></div>
         <div  class="nav-links nav-link-hide">
-
+            
         </div>
+
     </nav>
-    <section id="aboutMain" class="about-main">
+    <div id="main">
+        <!-- <section id="heroSec" class="hero-sec">
+
+
+        </section>
+        <section id="programDtls" class="program-dtls">
+
+
+        </section> -->
+        <section id="aboutMain" class="about-dtls">
+
+
+
+        </section>
+
+        <section class="about-logo-sec">
+            <div class="about-logo-sec-txt">
+                <h2>GROWTH INTERST <br> GLOBAL SUMMIT <br> 2022</h2>
+                <hr>
+                <p>Market-facing ladder up / ladder back to the strategy build on a culture of contribution and inclusion.</p>
+                <div class="about-logo-img-cont">
+                    <img src="./res/footer-logo.svg" alt="about logo">
+                </div>
+            </div>
+        </section>
+
+        <section class="past-summits">
+            <div class="past-summits-txt">
+                <h3>GROWTH INTERST <br> GLOBAL SUMMIT <br> 2022 Logo</h3>
+                <hr>
+                <p>The website doesn't have the theme i was going for I really like the colour but can you change it there are more projects lined up charge extra the next time.</p>
+            </div>
+            <div class="past-summit-img-cont">
+                <div class="past-summit-img1 past-summit-img">
+                    <img src="./res/aboutgroup1.png" alt="past summit img1">
+                </div>
+                <div class="past-summit-img1 past-summit-img">
+                    <img src="./res/aboutgroup2.png" alt="past summit img1">
+                </div>
+                <div class="past-summit-img2"></div>
+            </div>
+        </section>
+
+
+
+        <section id="partnerDtls" class="partner-dtls">
+            
+
+        </section>
+
+        <footer id="footer">
+            <div class="footer-items">
+                <div id="footer-logo">
+                    <img src="./res/footer-logo.svg" alt="logo">
+                </div>
+                <div class="footer-text"><p> &copy; Your project so that you have <br> something unique in your portfolio.</p></div>
+            </div>
+
+        </footer>
```

Comment:
- [ ] The footer for the about page has black background color, but yours is white. Kindly change your background color from white to black